samples/detect.js

```diff
@@ -653,7 +653,7 @@ async function detectPdfText(bucketName, fileName) {
   // const fileName = 'path/to/document.pdf';
 
   const gcsSourceUri = `gs://${bucketName}/${fileName}`;
-  const gcsDestinationUri = `gs://${bucketName}/${fileName}.json`;
+  const gcsDestinationUri = `gs://${bucketName}/${fileName}/`;
```

Comment:
Maybe we should change this to `gcsDestinationUriPrefix`, to help suggest files are going to end up in a folder.

Also, the default value we provide for `fileName` is `path/to/document.pdf` (line 653). That would mean the results end up in `gs://my-bucket/path/to/document.pdf/`. I'm not sure what happens in that case-- does GCS overwrite the original file and replace it with the directory?

As an idea, we could do something like:

```js
const gcsDestinationUriPrefix = `gs://${bucketName}/DetectionResults/${fileName}/`;
// "gs://my-bucket/DetectionResults/path/to/document.pdf/"
```